fixtures/views/namespace.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: mission-control.flanksource.com/v1
+kind: View
+metadata:
+  name: namespace
+  namespace: mc
+spec:
+  columns:
+    - cardPosition: title
+      name: name
+      primaryKey: true
+      type: config_item
+      url:
+        config: row.id
+    - cardPosition: deck
+      filter:
+        type: multiselect
+      icon: row.health
+      name: status
+      type: status
+    - filter:
+        type: multiselect
+      hidden: true
+      name: health
+      type: health
+    - cardPosition: deck
+      description: Memory usage
+      gauge:
+        max: row.memory_limit
+        thresholds:
+          - color: green
+            percent: 0
+          - color: orange
+            percent: 75
+          - color: red
+            percent: 90
+      name: memory
+      type: gauge
+      unit: bytes
+    - cardPosition: deck
+      gauge:
+        max: row.cpu_limit
+        thresholds:
+          - color: "#8BC34A"
+            percent: 0
+          - color: "#F4B23C"
+            percent: 70
+          - color: "#F25C54"
+            percent: 85
+      name: cpu
+      type: gauge
+      unit: millicore
+    - cardPosition: footer
+      name: updated
+      type: datetime
+    - cardPosition: footer
+      name: created
+      type: datetime
+    - cardPosition: body
+      hidden: true
+      name: image
+      type: string
+    - cardPosition: body
+      hidden: true
+      name: node
+      type: string
+  description: View for inspecting a Kubernetes Namespace
+  display:
+    icon: namespace
+    plugins:
+      - configTab:
+          types:
+            - Kubernetes::Namespace
+        variables:
+          cluster: $(.config.tags.cluster)
+          namespace: $(.config.name)
```

Comment:
This is the new addition.

The frontend makes a call to the API to get these variables. Which it uses to load the view with appropriate config.